cmd/minikube/cmd/config/config.go

```diff
@@ -143,12 +148,12 @@ func ReadConfig() (MinikubeConfig, error) {
 func WriteConfig(m MinikubeConfig) error {
 	f, err := os.Create(constants.ConfigFile)
 	if err != nil {
-		return fmt.Errorf("Could not open file %s: %s", constants.ConfigFile, err)
+		return fmt.Errorf("Cannot open file %s: %s", constants.ConfigFile, err)
 	}
 	defer f.Close()
 	err = encode(f, m)
 	if err != nil {
-		return fmt.Errorf("Error encoding config %s: %s", constants.ConfigFile, err)
+		return fmt.Errorf("Cannot write config %s: %s", constants.ConfigFile, err)
```

Comment:
encoding means changing something from one format to another, while in this context the error message for encoding and decoding look like messages for getting/setting. please correct me if I'm off here, or explain what encoding means in this context (I've never seen encode/decode used in a config context)